<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetScalarField</class>
 <widget class="QWidget" name="WidgetScalarField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scalar Field</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxScalarFieldName">
       <property name="toolTip">
        <string>Select the scalar field to visualize</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Edge Style</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditScalarFieldMinValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderScalarFieldMaxValue">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBoxScalarFieldAutoRange">
       <property name="toolTip">
        <string>Normalize the scalar field to [-1, 1]</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Use Texture</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEditScalarFieldMaxValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxScalarFieldUseTexture">
       <property name="toolTip">
        <string>Use texture for color mapping</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxScalarFieldEdgeStyle">
       <property name="toolTip">
        <string>Styles of scalar fields defined on edges</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBoxScalarFieldTexture">
       <property name="toolTip">
        <string>Colormap style</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Auto Range</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderScalarFieldMinValue">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxScalarFieldName</tabstop>
  <tabstop>comboBoxScalarFieldEdgeStyle</tabstop>
  <tabstop>checkBoxScalarFieldUseTexture</tabstop>
  <tabstop>comboBoxScalarFieldTexture</tabstop>
  <tabstop>checkBoxScalarFieldAutoRange</tabstop>
  <tabstop>lineEditScalarFieldMinValue</tabstop>
  <tabstop>horizontalSliderScalarFieldMinValue</tabstop>
  <tabstop>lineEditScalarFieldMaxValue</tabstop>
  <tabstop>horizontalSliderScalarFieldMaxValue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
